api/src/main/java/io/strimzi/api/kafka/model/connector/AbstractConnectorSpec.java

```diff
@@ -134,4 +137,38 @@ public ConnectorState getState() {
     public void setState(ConnectorState state) {
         this.state = state;
     }
+
+    /**
+     * @return The listOffsets configuration.
+     */
+    @Description("Configuration for listing offsets")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public ListOffsets getListOffsets() {
+        return listOffsets;
+    }
+
+    /**
+     * Configures the behaviour when listing connector offsets.
```

Comment:
what does exactly "behaviour" mean here?

Reply:
I picked the term "behaviour" as a vague term so that in future if we added different options within ListOffsets it would be covered. So "behaviour" today means where we write the offsets to, but in future could reference any kind of configuration that changes what the operator does when performing the list offsets action, for example writing the output to somewhere else, or even if we made it possible to change e.g. what partitions it lists offsets for.

Reply:
Should I change it to something like "Sets the configuration for listing connector offsets"?

Reply:
I think that "configuration" is better, "behaviour" sounds really too vague to me (so I didn't get what you meant).